pkg/api/types.go

```diff
@@ -403,6 +404,13 @@ const (
 	PodUnknown PodPhase = "Unknown"
 )
 
+type PodCondition string
+
+// These are the valid service statuses of pods.
+const (
+	PodInService = "InService"
```

Comment:
if a pod can only be InService, is this useful?  Shouldn't there be a second condition for this?